assets/js/components/GoogleChartV2.js

```diff
@@ -147,6 +147,7 @@ export default function GoogleChartV2( props ) {
 				className="googlesitekit-chart-v2__inner"
 				chartEvents={ combinedChartEvents }
 				chartType={ chartType }
+				chartVersion={ 49 }
```

Comment:
Nit-pick, but shouldn't this be a string? Not a big deal, but it _is_ a string in the regular JS snippet that we use in PHP, and also typically a string in the React wrapper codebase: https://github.com/rakannimer/react-google-charts/blob/master/src/types.ts#L257